test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex routing url
+   Given Minishift has state "Running"
+     When status code of HTTP request to "Routing" is equal to "200"
```

Comment:
Agree. In this case the step definition uses ```getRoutingUrl()``` and the constant ```get route -o=custom-columns=URL:.spec.host``` to get the route of deployed application. As ruby-ex is the only app deployed in this scenario, we wont get any problem in this case.  But its good idea to pass the argument to make it more specific.